packages/markdown/src/lib/MarkdownPlugin.ts

```diff
@@ -43,6 +54,7 @@ export const MarkdownPlugin = createTSlatePlugin<MarkdownConfig>({
   options: {
     elementRules: remarkDefaultElementRules,
     indentList: false,
+    keepLineBreak: false,
```

Comment:
This should be the opposite default. So I would name it `splitNewLines: false`

Reply:
@zbeyens Sorry, I didn't know you were reviewing at the same time 😄